serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -1 +1,274 @@
-= Create your first workflow service
\ No newline at end of file
+= Create your first workflow service
+
+Learn how to create a Hello World Serverless Workflow application. This guide covers:
+
+* Bootstrapping an application
+* Creating a workflow
+* Running the application
+* Testing the application
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* https://adoptium.net/[JDK 11+] installed with `JAVA_HOME` configured appropriately
+* https://maven.apache.org/install.html[Apache Maven]
+* https://quarkus.io/guides/cli-tooling[Quarkus CLI]
+
+To edit your workflows:
+
+* Visual Studio Code with https://marketplace.visualstudio.com/items?itemName=redhat.java[Red Hat Java Extension]
+and https://marketplace.visualstudio.com/items?itemName=redhat.vscode-extension-serverless-workflow-editor[Red Hat Serverless Workflow Editor]
+installed
+
+== Architecture
+
+In this guide, we'll create a Serverless Workflow application that serves a `hello_world` endpoint.
+The workflow comprises two states:
+
+. `Inject Hello World` - Injects a `Hello World` message into the response
+. `Inject Mantra` - Injects a `Mantra` message into the response
+
+image::getting-started/hello-world-workflow.png[]
+
+== Solution
+We recommend that you follow the instructions from <<boostrapping-the-project,Bootstrapping the Project>>
+and onwards to create the application step by step.
+
+However, you can go right to the completed example by cloning the git repository:
+
+The solution is located in the `kogito-quarkus-examples/serverless-workflow-hello-world`
+https://github.com/hbelmiro/kogito-examples/tree/main/kogito-quarkus-examples/serverless-workflow-hello-world[directory].
```

Comment:
This link is broken and redirects to a downstream repo. This should refer to upstream `kogito-example` repo.

Reply:
Good catch regarding the downstream repo :)
The link is broken because it still doesn't exist. It's supposed to be created in https://github.com/kiegroup/kogito-examples/pull/1266, which is a dependency of this PR.
Anyway, the link is going to change because a new repo will be created for the guides.